src/plugins/usage_collection/server/routes/stats/README.md

```diff
@@ -1,6 +1,6 @@
 # `/api/stats`
 
-This API returns the metrics for the OpenSearch Dashboards server and usage stats. It allows the [Metricbeat OpenSearch Dashboards module](https://www.opensearch.org/guide/en/beats/metricbeat/current/metricbeat-module-kibana.html) to collect the [stats metricset](https://www.opensearch.org/guide/en/beats/metricbeat/current/metricbeat-metricset-kibana-stats.html).
+This API returns the metrics for the OpenSearch Dashboards server and usage stats. It allows the [Metricbeat OpenSearch Dashboards module](https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-module-kibana.html) to collect the [stats metricset](https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-metricset-kibana-stats.html).
```

Comment:
We should probably remove these links as well.

Reply:
Regarding the `beats*` module related stuff. There still are some code references in codebase. We probably need a separate PR to do these if we really need to clean them up.

Reply:
I think the references in the code are fine for now, but we can route `Metricbeat OpenSearch Dashboards module` to `https://opensearch.org/docs/latest/clients/agents-and-ingestion-tools/index/` and then remove the link for `stats metricset`.